### PR TITLE
Feature tea changes

### PR DESCRIPTION
**Pull request by nareshkumarthota**
This PR implements feature #331

### CODE REVIEW

**Comment by ykalidin** [comment]
Working fine with latest changes. All app ,rms and tea docker images are working as expected.

File: cloud/docker/build_image.sh [hunk 1]
```diff
@@ -497,6 +502,34 @@ if [ "$INSTALLATION_TYPE" = "fromlocal" ]; then
         fi
     fi
 
+    if [ $(echo "${ARG_BE_VERSION//.}") -ge 630 -a  "$IMAGE_NAME" = "$TEA_IMAGE" ]; then
+        # get tea details
+        TEA_HOME=$(cat $BE_HOME/$TRA_FILE | grep ^tibco.env.TEA_HOME | cut -d'=' -f 2 | sed -e 's/\r$//' )
+        if [ "$TEA_HOME" = "" ]; then
+            echo "ERROR: TEA_HOME is not set in tea tra file. Please check"
```
Comment (rameshpolishetti) [inline cloud/docker/build_image.sh:509]: would you like to change tea -> TEA

Comment (nareshkumarthota) [reply cloud/docker/build_image.sh]: Done

**Review by rameshpolishetti — APPROVED** [review]
Looks good to me.

**Review by rameshpolishetti — APPROVED** [review]
looks good.

**Comment by ykalidin** [comment]
-  TEA server-2.4.0 and BE(TEA agent ) - 6.2.2 and BE Fraud Detection example (6.2.2,6.3)
-  TEA server-2.4.1 and BE(TEA agent ) - 6.3 and BE Fraud Detection example (6.2.2,6.3)

Verified the above scenarios with Installers and local installation in linux and windows-linux containers in ECS and ECS Fargate.

Verified the same with Installers and local installation in linux containers in EKS and EKS Fargate.

Verified 6.3 Fraud Detection cache example in ECS.
All are working as expected.

**Pull request merged**